Docs/Solar_Battery_Power/readme.md

```diff
@@ -42,10 +45,19 @@ This design has three major circuits:
 
 ### Schematic
 
-![Schematic](v1.d/Schematic_v1.d.svg)
+![Schematic](v2.b/Schematic_v2.b.svg)
 
 ## Release Notes
 
+## v2.b
+
+* added diode after power supply to not back-power 6V_Solar 
+* pull `BATT_DISABLE` off of the solar ADC circuit to not affect ADC readings
+
+## v2.a
+
+* BREAKING CHANGE: added `BATT_DISABLE` and changed `VOUT` to `V_MAIN` to ensure external battery control and not overvoltage the battery
```

Comment:
Can you explain that this has a breaking pin change please? Also, **please bold the breaking change** bit.